pom.xml

```diff
@@ -105,4 +100,9 @@
             <version>1.2.6</version>
         </dependency>
     </dependencies>
+
+    <properties>
+        <sling.java.version>8</sling.java.version>
+        <project.build.outputTimestamp>1</project.build.outputTimestamp>
```

Comment:
This does not work due to https://issues.apache.org/jira/browse/MJAVADOC-684. Instead use a timestamp like this: <project.build.outputTimestamp>2021-08-06T09:01:40Z</project.build.outputTimestamp>. The value does not matter as it is updated automatically during release:prepare.

Reply:
Fixed.